src/lib/prisma.ts

```diff
@@ -1,12 +1,12 @@
 import { PrismaClient } from "@prisma/client";
 
-// Prevent multiple instances of Prisma Client in development
 declare global {
+  // Only var creates properties on the global object, used to prevent multiple Prisma instances in dev
+  // eslint-disable-next-line no-var
   var prisma: PrismaClient | undefined;
 }
 
 const prisma = global.prisma || new PrismaClient();
-
 if (process.env.NODE_ENV !== "production") {
   global.prisma = prisma;
 }
```

Comment:
This was a rare situation where the outdated var worked correctly here... for some reason.
Solution? Tell nextjs not to care about the var keyword here.